### ISSUE TITLE
Options not in header error for EP071999

### ISSUE THREAD

**Issue by timtrice** [ropensci/rrricanes#56]
Error:

```
Error in if (!any(stringr::str_count(header, paste(options, sep = "|")))) stop(sprintf("Options not in header. %s",  : 
  missing value where TRUE/FALSE needed 
```

Traceback:

```
11.
scrape_status(header) at scrapers.R#225
10.
scrape_header(contents, ret = "status") at fstadv.R#207
9.
.f(.x[[i]], ...) 
8.
purrr::map(filter_fstadv(products), fstadv) at fstadv.R#157
7.
get_fstadv(url) at check-errors.R#33
6.
.f(.x[[i]], ...) 
5.
map(.x = nums, .f = function(x) {
    url <- storms %>% slice(x) %>% .$Link
    key <- storms %>% slice(x) %>% .$Key
    obs.fstadv <- get_fstadv(url) ... at check-errors.R#30
4.
eval(ei, envir) 
3.
eval(ei, envir) 
2.
withVisible(eval(ei, envir)) 
1.
source("~/Projects/Hurricanes/data-raw/check-errors.R") 
```

**Comment by timtrice** [comment]
[Advisory 13](http://www.nhc.noaa.gov/archive/1999/mar/MEP0799.013.html) contains carriage returns (\r) in addition to newlines (\n). 

Here is an example from [Advisory 12](http://www.nhc.noaa.gov/archive/1999/mar/MEP0799.012.html) which parses correctly:

```
\n\nZCZC MIATCMEP2 ALL\nTTAA00 KNHC DDHHMM\nHURRICANE DORA FORECAST/ADVISORY NUMBER  12\nNATIONAL WEATHER SERVICE MIAMI FL   EP0799\n2100Z SUN AUG 08 1999\n \n
```

And [Advisory 13](http://www.nhc.noaa.gov/archive/1999/mar/MEP0799.013.html), which generates the error:

```
\n\nZCZC MIATCMEP2 ALL\r\nTTAA00 KNHC DDHHMM\r\nHURRICANE DORA FORECAST/ADVISORY NUMBER  13\r\nNATIONAL WEATHER SERVICE MIAMI FL   EP0799\r\n0300Z MON AUG 09 1999\r\n \r\n
```

Patch must allow for either/or \n and \r.

**Comment by timtrice** [comment]
Resolved in commit 7f742a6.
